### PR TITLE
Php8 Support

### PR DESCRIPTION
**Pull request by klodoma**
- add php8 support
- update composer dependencies
- drop support for older php versions

### CODE REVIEW

**Comment by klodoma** [comment]
Please have a look at these changes. I had a PHP8 dependency so I had to dig in a bit more.

File: composer.json [hunk 1]
```diff
@@ -3,22 +3,27 @@
...
+    "authors": [
+        {
+            "name": "Kevin Saliou",
+            "email": "kevin@saliou.name",
+            "homepage": "http://kevin.saliou.name"
+        }
+    ],
     "require": {
-        "php": "^5.4 || ^7.2",
+        "php": "^5.4 || ^7.2 || ^8.0",
```
Comment (kbsali) [inline composer.json:19]:
```suggestion
        "php": "^7.3 || ^8.0",
```
right?

**Review by kbsali — COMMENTED** [review]
Awesome! :+1: 
What do you think about the version to be tagged once this is merged?
v1.6.0? or v2.0.0 ? There is no major change to the API itself, but to the dependencies yes! :)

File: composer.json (hunk 1 above)
Comment (klodoma) [reply composer.json:19]: Yes, that's right. It won't work on 7.2

**Comment by klodoma** [comment]
v2.0.0 sounds good. I think it makes sense. Practically we'll kill Support for Php < 7.3 but that's fine I guess.

File: composer.json (hunk 1 above)
Comment (kbsali) [reply composer.json:19]: could you commit the suggestion?

Comment (klodoma) [reply composer.json]: Sure. Sorry, I've missed it 👍

**Comment by Art4** [comment]
Please: We should go for v1.6.0. Removing support for PHP versions is not a breaking change. You are only creating work for devs who already migrated to PHP 7.3+ or PHP 8.

If you go for v2, everyone has manually change their dependencies without a reason (because there are no breaking changes), but everyone who is on PHP <7.2 don't have to do anything, because they couldn't update anyway. It should be the other way.

**Comment by klodoma** [comment]
Please release asap, so I can move away from my fork. Thank you!

**Comment by kbsali** [comment]
@Art4 thanks for your insight! You convinced me to release it as 1.6.0! :+1:

**Comment by kbsali** [comment]
@klodoma I have just pushed the new tag (sorry, I was away with no connection until today)

**Pull request merged**